### PR TITLE
Rep 2669

### PR DESCRIPTION
**Pull request by JoelRizner**

### CODE REVIEW

File: repose-aggregator/artifacts/src/config/filters/response-messaging.cfg.xml [hunk 1]
```diff
@@ -4,30 +4,15 @@
...
-            {
-            "overLimit" : {
-            "code" : 413,
-            "message" : "OverLimit Retry...",
-            "details" : "Error Details...",
-            "retryAfter" : "%{Retry-After DATE ISO_8601}o"
-            }
-            }
+    <status-code id="418" code-regex="418">
+        <message media-type="/" content-type="application/json">
```
Comment (dmnjohns) [inline repose-aggregator/artifacts/src/config/filters/response-messaging.cfg.xml]: Should the `media-type` here be set to `*/*`? What exactly does `/` match?

File: repose-aggregator/artifacts/src/config/filters/response-messaging.cfg.xml [hunk 2]
```diff
@@ -4,30 +4,15 @@
...
             <![CDATA[
-<overLimit
-    xmlns="http://docs.openstack.org/compute/api/v1.1"
-    code="413"
-    retryAfter="%{Retry-After DATE ISO_8601}o">
-  <message>OverLimit Retry...</message>
-  <details>Error Details...</details>
-</overLimit>
+<I> am a teapot</I>
 ]]>
```
Comment (dmnjohns) [inline repose-aggregator/artifacts/src/config/filters/response-messaging.cfg.xml]: Formatting on this block is odd. I would just single-line it?

**Comment by dmnjohns** [comment]
LGTM.

👍

**Comment by adrianjgeorge** [comment]
Looks fine

**Comment by adrianjgeorge** [comment]
retest this please

**Comment by dmnjohns** [comment]
retest this please

**Comment by Mario-Lopez** [comment]
I was thinking we could merge this already since the failure should be fixed on master now.

**Comment by dmnjohns** [comment]
I was just going to let the tests one through once more to verify, because why not?

**Comment by Mario-Lopez** [comment]
I was going to say we should do a rebase if we wanted to re-run the test, but I forgot it builds a merged version anyway.  👍  to another test run.

**Comment by Mario-Lopez** [comment]
yay, build passed

**Comment by dmnjohns** [comment]
Merging.

**Pull request merged**